demo/src/app/components/+dropdown/dropdown-section.list.ts

```diff
@@ -19,6 +19,7 @@ import {
   NgApiDocComponent,
   NgApiDocConfigComponent
 } from '../../docs/api-docs';
+import {DemoDropdownCustomHtmlComponent} from "./demos/custom-html/custom-html";
```

Comment:
configure your IDE to format import lines according to our style guide
`import { DemoDropdownCustomHtmlComponent } from './demos/custom-html/custom-html';`